index/views.py

```diff
@@ -86,14 +86,17 @@ def get_list(self):
 Checks if game is player's library and if it is it return False else return True
 """
 
+class LibraryDelete(View):
+    def get(self, request, game_id, **kwargs):
+        player_library = Library_Model.objects.filter(
+            owner_id=self.request.user.id
+        )
+        print(player_library)
+        game = Library_Membership.objects.filter(library=player_library[0], game=game_id)
+        print(game)
+        game.delete()
+        return HttpResponseRedirect(reverse("library"))
 
-def checkLibraryForGame(user_id, game_id):
-    player_library = Library_Model.objects.filter(
-        owner_id=user_id
-    )
-    game = player_library[0].games.filter(game_id=game_id)
-    game_available = True if game.count() == 0 else False
-    return game_available
```

Comment:
This is what cause the conflict, i just deleted the version from paginator and kept the one from library